Release-Notes-R5.md

```diff
@@ -2,38 +2,22 @@
-
-### cert-manager 1.12.x
-
-### kind 0.18.0
-
-### helm 3.12.x
-
-### metrics-server 0.6.4
-
-### nginx-ingress 1.8.x
-
-- We removed support for older k8s versions <= 1.19. Ingress-nginx 1.8.1 supports k8s version >= 1.24.
-  See also
-  ingress-nginx [supported versions table](https://github.com/kubernetes/ingress-nginx#supported-versions-table).
-
-### k9s 0.27.x
-
-- In the previous releases, the latest version was used. Now, it is pinned.
-
-### calico 3.26.x
+| Software       | Version | Note                                                                                                                                                                            |
+|----------------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Terraform      | v1.4.6  | [#515](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/515) Given Hashicorp change of license to BSL we pinned terraform to version < 1.6.0.               |
+| flux2          | 0.41.x  |                                                                                                                                                                                 |
+| sonobuoy       | 0.56.x  | v0.56.17 adds support for the latest k8s versions even for k8s 1.27.                                                                                                            |
+| cilium         | 1.14.0  |                                                                                                                                                                                 |
+| cert-manager   | 1.12.x  |                                                                                                                                                                                 |
+| kind           | 0.18.0  |                                                                                                                                                                                 |
+| helm           | 3.12.x  |                                                                                                                                                                                 |
+| metrics-server | 0.6.4   |                                                                                                                                                                                 |
+| nginx-ingress  | 1.8.x   | Supports only k8s version >= 1.24. We dropped support for older k8s versions. [supported versions table](https://github.com/kubernetes/ingress-nginx#supported-versions-table). |
+| k9s            | 0.27.x  | Instead of latest version, it has been pinned.                                                                                                                                  |
```

Comment:
```suggestion
| k9s            | 0.27.x  | Instead of using just the latest version, it has been pinned now.                                                                                                                              |
```